packages/backend/src/core/discovery/proxies/GnosisSafe.ts

```diff
@@ -0,0 +1,66 @@
+import { providers } from 'ethers'
+
+import { ContractParameters } from '../types'
+import { bytes32ToAddress } from '../utils/address'
+import { getCallResult } from '../utils/getCallResult'
+import { getStorage } from '../utils/getStorage'
+import { GnosisSafe__factory } from './typechain/GnosisSafe__factory'
```

Comment:
I think I would rewrite this file to not use typechain and just declare the abi on the top of the file and the call and cast the values from any to whatever is needed. Yes it is less type safe, but we would also not have to commit autogenerated files from a dependency that is no longer used.